Files/ViewModels/ItemViewModel.cs

```diff
@@ -729,7 +729,9 @@ private async Task LoadItemThumbnail(ListedItem item, uint thumbnailSize = 20, I
                         var matchingStorageFile = (StorageFile)matchingStorageItem ?? await GetFileFromPathAsync(item.ItemPath);
                         if (matchingStorageFile != null)
                         {
-                            using var Thumbnail = await matchingStorageFile.GetThumbnailAsync(ThumbnailMode.ListView, thumbnailSize, ThumbnailOptions.ResizeThumbnail);
+                            var mode = thumbnailSize < 80 ? ThumbnailMode.ListView : ThumbnailMode.SingleItem;
+
+                            using var Thumbnail = await matchingStorageFile.GetThumbnailAsync(mode, thumbnailSize, ThumbnailOptions.UseCurrentScale);
```

Comment:
Should this be done also on line 805 (when loading a folder thumbnail)?

Reply:
@gave92 I also wonder whether my addition of `ThumbnailOptions.ReturnOnlyIfCached` on that line has something to do with it?

Reply:
oddly, this seems to break thumbnails randomly
![image](https://user-images.githubusercontent.com/59544401/128639897-26fa1d89-751e-470a-aba8-9b73a90526a7.png)